net-firewall/ferm/ferm-2.3.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils systemd
```

Comment:
I don't think any of the functionality of eutils is being utilized, is it necessary?